bin.src/atoods

```diff
@@ -23,14 +23,22 @@
 
 import asyncio
 import logging
+import os
 
 import lsst.log as lsstlog
 from lsst.ctrl.oods.atoods_csc import AtOodsCsc
 
 lsstlog.usePythonLogging()
 
+oods_level = "INFO"
+if "CTRL_OODS_DEBUG_LEVEL" in os.environ:
+    oods_level = os.environ["CTRL_OODS_DEBUG_LEVEL"]
+
+level_mapping = logging.getLevelNamesMapping()
+debug_level = level_mapping.get(oods_level, logging.INFO)
+
```

Comment:
Similarly to my earlier point, I think replacing `oods_level` and `debug_level with, respectively, `log_level_name` and `log_level` would make following this snippet more easy.

Also, it would be nice to encapsulate this logic in a function so you could reuse it where it is needed.